server/app/routes/users/index.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const router = require('express').Router();
+const mongoose = require('mongoose');
+const Users = mongoose.model('Users');
+const Reviews = mongoose.model('Reviews');
+module.exports = router;
+
+router.get('/', (req, res, next) => {
+	Users.find({})
+		.then(users => res.json(users))
+		.catch(next);
+});
+
+router.post('/', (req, res, next) => {
+	Users.findOrCreate(req.body)
```

Comment:
mongoose doesn't have a findOrCreate, we either need a static to do this or a module